certbot/docs/contributing.rst

```diff
@@ -42,12 +42,12 @@ Install and configure the OS system dependencies required to run Certbot.
    # For APT-based distributions (e.g. Debian, Ubuntu ...)
    sudo apt update
    sudo apt install python3-dev python3-venv gcc libaugeas0 libssl-dev \
-                    libffi-dev ca-certificates openssl
+                    libffi-dev ca-certificates openssl pkg-config
    # For RPM-based distributions (e.g. Fedora, CentOS ...)
    # NB1: old distributions will use yum instead of dnf
    # NB2: RHEL-based distributions use python3X-devel instead of python3-devel (e.g. python36-devel)
    sudo dnf install python3-devel gcc augeas-libs openssl-devel libffi-devel \
-                    redhat-rpm-config ca-certificates openssl
+                    redhat-rpm-config ca-certificates openssl pkg-config
```

Comment:
> If I understand correctly, we are mentioning pkg-config here because of the possibility that a cryptography wheel is not available on their platform.

That's correct. That's actually what many of these dependencies are listed for and it just looks like we never updated them after cryptography added the rust dependency.

I'm not 100% sure all tests will pass with the changes I just made, but if they do, how about something like that?